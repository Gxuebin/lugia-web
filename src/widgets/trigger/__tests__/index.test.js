//@flow
import React from 'react';
import chai from 'chai';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Trigger from '../';

const { expect: exp, } = chai;

describe('trigger', () => {

  it('render', () => {
    expect(renderer.create(<Trigger><span>hello world!</span></Trigger>).toJSON()).toMatchSnapshot();
  });


});
