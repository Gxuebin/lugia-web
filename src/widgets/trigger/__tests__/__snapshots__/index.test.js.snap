// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trigger render 1`] = `
<span
  onBlur={undefined}
  onClick={undefined}
  onFocus={undefined}
  onMouseDown={undefined}
  onMouseEnter={undefined}
  onMouseLeave={undefined}
  onTouchStart={undefined}
>
  hello world!
</span>
`;
